examples/mnist/README.md

```diff
@@ -13,7 +13,7 @@ Then you can open jupyter notebook at [`http://localhost:8888`](http://localhost
 ## Quick start (GPU)
 
 ```bash
-$ envd up -f build_gpu.envd
+$ envd up -f :build_gpu # Call build_gpu function in build.envd
```

Comment:
We also need to add the doc for the change. Please open an issue to keep track of the doc change.

Reply:
at #547